binder.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: page_md
+title: The Binder Project
+title_image: assets/logo_binder.svg
+tagline: Reproducible, sharable, interactive computing environments
+permalink: /binder
+---
+
+## What is the Binder Project?
+
+The Binder project brings the power of customisable and easy to share computing
+environments to everyone. It allows users to specify custom environments,
+automates their creation, and allows users to these environments with a single link.
+Use-cases involve workshops, scientific workflows and fast sharing of work in teams.
+
+The Binder Project builds tools that reward best-practices in reproducible data
+science by piggy-backing off of community-developed standards for
+reproducibility. If repositories follow these best-practices and are hosted in
+an online repository, then Binder will build the environment needed to run the
+code and provide a link that allows others to interact with the results and
+reproduce analyses.
+
```

Comment:
Overall this intro is a bit verbose. I don't know exactly who your target audience is, but as a general rule I would not assume casual observers would read this much text.

Reply:
So I'm also offering something helpful here's a shortened version:
"The Binder project offers an easy place to share computing environments to everyone. It allows users to specify custom environments and share them with a single link. Use-cases involve workshops, scientific workflows and streamline sharing among teams."

The Binder Project builds tools that reward best-practices in reproducible data
science by utilizing community-developed standards for
reproducibility. When repositories follow these best-practices and are hosted in
an online repository, then Binder automatically builds a linkable environment anybody can access."

Reply:
just added in this language verbatim - nice!